### PR TITLE
feat: new sponsor attributes

### PR DESCRIPTION
**Pull request by JustinCast**
# Support new sponsor attributes
This PR support new necessary sponsor attributes

### CODE REVIEW

File: services/bios/genesis.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "initial_timestamp": "2020-08-11T04:20:00.000",
-  "initial_key": "EOS7GY6pQ86iWNS3DtS15gQ96ZNoyoNr9Ca7Y74mqUDmWunmbz9DP",
```
Comment (xavier506) [inline services/bios/genesis.json:3]: @JustinCast  can we keep the original key? let me know and i can send you the private key for it

File: services/bios/lifebank/lifebankcode/lifebankcode.abi [hunk 2]
```diff
@@ -325,42 +345,42 @@
         {
             "name": "adddonor",
             "type": "adddonor",
-            "ricardian_contract": "---\nspec-version: 0.1.0\ntitle: Add Donor\nsummary: The intent of the `{{ adddonor }}` action is to register a new donor in a community.\nicon: https://lifebank.io/icons/multisig.png"
```
Comment (xavier506) [inline services/bios/lifebank/lifebankcode/lifebankcode.abi]: @JustinCast  seems we are replacing the ricardian clauses with blank strings

File: services/bios/genesis.json (hunk 1 above)
Comment (JustinCast) [reply services/bios/genesis.json:3]: Yeah, why not

File: services/bios/lifebank/lifebankcode/lifebankcode.abi [hunk 3]
```diff
@@ -400,27 +420,6 @@
             "key_types": []
         }
     ],
-    "ricardian_clauses": [
```
Comment (xavier506) [inline services/bios/lifebank/lifebankcode/lifebankcode.abi]: @JustinCast  lest keep the ricardian clauses and republish the contracts

File: services/bios/utils/lifebank.sh [hunk 4]
```diff
@@ -161,12 +161,17 @@ register_sponsor() {
         "covid_impact":"Los clientes le huyen al covid",
         "benefit_description":"10% off toilet seats",
         "website":"https://garberhardware.com/",
-        "telephone":"(212) 206 6996",
-        "bussines_type":"Construction",
+        "telephones":"[\"134123\", \"09090032\"]",
```
Comment (xavier506) [inline services/bios/utils/lifebank.sh:164]: where is this phone format from?

Comment (JustinCast) [reply services/bios/utils/lifebank.sh:164]: The phone format needs to be done in the frontend or in the backend. I just wrote some placeholder text

**Review by xavier506 — APPROVED** [review]

**Pull request merged**